common/culture/innovations/wc_cultural_maa_innovations.txt

```diff
@@ -38,10 +99,847 @@ innovation_taming_the_skies = {
+		OR = {
+			this = culture:tirassian
+			this = culture:baradin
+		}
+	}
+
+	unlock_maa = baradian_marine
+
+	flag = global_maa
+}
+
+
+innovation_blackrock_legionnaire = {
+	group = culture_group_regional
+	culture_era = culture_era_early_medieval
+	icon = @legionnaire
+
+	potential = {
+		this = culture:blackrock
+	}
+
+	unlock_maa = blackrock_legionnaire
+
+	flag = global_maa
+}
+
+innovation_chaosborn = {
+	group = culture_group_regional
+	culture_era = culture_era_early_medieval
+	icon = @chaosborn
+
+	potential = {
+		OR = {
+			this = culture:eredar
+			this = culture:eredruin
+		}
+	}
+
+	unlock_maa = chaosborn
+
+	flag = global_maa
+}
+
+innovation_high_blood_elf_combo = {
+	group = culture_group_regional
+	culture_era = culture_era_early_medieval
+	icon = @spellbreaker
+
+	potential = {
+		OR = {
+			this = culture:high_elf
+			this = culture:blood_elf
+		}
+	}
+
+	unlock_maa = farstriders
+	unlock_maa = dragonhawk_rider
+	unlock_maa = spellbreaker
+
+	flag = global_maa
+}
+
+innovation_fel_guard = {
+	group = culture_group_regional
+	culture_era = culture_era_early_medieval
+	icon = @fel_guard
+
+	potential = {
+		OR = {
+			this = culture:annihilan
+			this = culture:nathrezim
+			this = culture:shivarra
+			this = culture:observer
+			this = culture:sayaadi
+			this = culture:satyr
+		}
+	}
+
+	unlock_maa = fel_guard
+
+	flag = global_maa
+}
+
+innovation_forsaken_ranged = {
+	group = culture_group_regional
+	culture_era = culture_era_early_medieval
+	icon = @dark_ranger
+
+	potential = {
+		this = culture:forsaken
+	}
+
+	unlock_maa = dark_ranger
+
+	flag = global_maa
+}
+
+innovation_golem_arcane = {
+	group = culture_group_regional
+	culture_era = culture_era_early_medieval
+	icon = @arcane_golem
+
+	potential = {
+		OR = {
+			this.culture_group = culture_group:draenei_group
+			this = culture:dalaranian
+		}
+	}
+
+	unlock_maa = golem_arcane
+
+	flag = global_maa
+}
+
+innovation_golem_mech = {
+	group = culture_group_regional
+	culture_era = culture_era_early_medieval
+	icon = @mech_golem
+
+	potential = {
+		this.culture_group = culture_group:dwarf_group
+	}
+
+	unlock_maa = golem_mech
+
+	flag = global_maa
+}
+
+innovation_gryphon = {
+	group = culture_group_regional
+	culture_era = culture_era_early_medieval
+	icon = @gryphon
+
+	potential = {
+		OR = {
+			this.culture_group = culture_group:human_group
+			this.culture_group = culture_group:dwarf_group
+		}
+	}
+
+	unlock_maa = gryphon_rider
+
+	flag = global_maa
+}
+
+innovation_night_elf_ground = {
+	group = culture_group_regional
+	culture_era = culture_era_early_medieval
+	icon = @watcher
+
+	potential = {
+		this = culture:night_elf
+	}
+
+	unlock_maa = watcher
+	unlock_maa = mountain_giant
+
+	flag = global_maa
+}
+
+innovation_pandaren_ground = {
+	group = culture_group_regional
+	culture_era = culture_era_early_medieval
+	icon = @wardancer
+
+	potential = {
+		this.culture_group = culture_group:pandaren_group
+	}
+
+	unlock_maa = bearpaws
+	unlock_maa = wardancer
```

Comment:
They had it from the first suggestion, dunno, I merely tweaked the stats for the rebalance...

Reply:
> They had it from the first suggestion, dunno, I merely tweaked the stats for the rebalance...

Mmmkay, my suggestion will be removing it.

Reply:
Should they get another terrain modifier then? hills or mountains maybe?

Reply:
> Should they get another terrain modifier then? hills or mountains maybe?

If you base it on Shado-pan, it may be a thing. Both hills and mountains then.